cmd/nodecmd/local.go

```diff
@@ -109,7 +109,12 @@ You can check the bootstrapping status by running avalanche node status local.
 	networkoptions.AddNetworkFlagsToCmd(cmd, &globalNetworkFlags, false, networkoptions.DefaultSupportedNetworkOptions)
 	cmd.Flags().BoolVar(&latestAvagoReleaseVersion, "latest-avalanchego-version", true, "install latest avalanchego release version on node/s")
 	cmd.Flags().BoolVar(&latestAvagoPreReleaseVersion, "latest-avalanchego-pre-release-version", false, "install latest avalanchego pre-release version on node/s")
-	cmd.Flags().StringVar(&useCustomAvalanchegoVersion, "custom-avalanchego-version", "", "install given avalanchego version on node/s")
+	cmd.Flags().StringVar(
+		&useCustomAvalanchegoVersion,
+		"custom-avalanchego-version",
+		constants.FujiAvalancheGoV113,
```

Comment:
this will break the fix i made today, lets undo this change and just change the constant